share/completions/diskutil.fish

```diff
@@ -0,0 +1,152 @@
+#completion for diskutil
+
+function __fish_diskutil_needs_command
+  set cmd (commandline -opc)
+  if [ (count $cmd) -eq 1 ]
+    return 0
+  end
+  return 1
+end
+
+function __fish_diskutil_using_command
+  set cmd (commandline -opc)
+  if [ (count $cmd) -gt 1 ]
+    if [ $argv[1] = $cmd[2] ]
+      return 0
+    end
+  end
+  return 1
+end
+
+function __fish_diskutil_devices
+    ls /dev/disk*
```

Comment:
Or remove the fork entirely by using builtins. Would something like `string replace -r '^.*/' '' -- /dev/disk*/*` work? Or how is the structure here? I'm assuming there's directories named "disk*"?